qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -58,7 +59,7 @@ def _redbiom_feature_search(self, query, contexts):
         study_artifacts = defaultdict(lambda: defaultdict(list))
         query = [f for f in query.split(' ')]
         for ctx in contexts:
-            for idx in redbiom.util.ids_from(query, True, 'feature', ctx):
+            for idx in redbiom.util.ids_from(query, False, 'feature', ctx):
```

Comment:
This explicit switch from True to False is intentional right?

Reply:
This is the main reason we need this change!